src/Molecules/Export/Export.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { css } from '@styled-system/css'
+import RadioGroup from '@material-ui/core/RadioGroup'
+import { Download } from 'styled-icons/boxicons-regular/Download'
+
+import Trans, { useTranslation } from '../../Atoms/Trans'
+import Radio from '../../Atoms/Radio'
+import FormControl, { FormLabel, FormControlLabel } from '../../Molecules/FormControl'
+import InputLabel from '../../Molecules/FormControl/InputLabel'
+import Input from '../../Atoms/Input'
+import Select from '../../Atoms/Select'
+import Typo from '../../Atoms/Typo'
+import Button from '../../Atoms/Button'
+
+const Body = styled.div`
+  display: flex;
+  flex-direction: column;
+  ${css({ px: 'xl' })}
+  width: 250px;
+
+  ${FormControl} {
+    ${css({ mt: 'xl' })}
+  }
+`
+
+const Actions = styled.div`
+  display: flex;
+  justify-content: space-between;
+  border-top: 1px solid;
+  ${css({
+    borderTopColor: 'secondary',
+    pt:             'l',
+    pb:             's',
+    px:             'l',
+    mt:             'l',
+  })}

```

Comment:
I did it this way for this component because I think it is not supposed to be overridable. This way we have visual coherence over all the uses of this one.